pkgs/applications/virtualization/ddev/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, ddev }:
+
+buildGoModule rec {
+  pname = "ddev";
+  version = "1.22.1";
+
+  src = fetchFromGitHub {
+    owner = "ddev";
+    repo = "ddev";
+    rev = "v${version}";
+    sha256 = "sha256-3uDdp827noiY6P7VSBXM34MTe9+MCEkNtuRQ3eFoBnk=";
+  };
+
+  vendorSha256 = null;
+
+  ldflags = [
+    "-extldflags -static"
+    "-X github.com/ddev/ddev/pkg/versionconstants.DdevVersion=${version}"
+    "-X github.com/ddev/ddev/pkg/versionconstants.SegmentKey=${version}"
+  ];
+
+  doCheck = false;
```

Explain why (in a comment):
```suggestion
  # <reason why>
  doCheck = false;
```